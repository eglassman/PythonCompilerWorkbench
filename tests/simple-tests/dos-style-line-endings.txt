# DOS-style line endings with \r\n to end each line
'''
x'''
