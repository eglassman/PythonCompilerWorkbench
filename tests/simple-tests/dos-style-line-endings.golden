# DOS-style line endings with \r\n to end each line
'''
x'''
===
Module
  body: [
    Expr
      value: Str ML>> "'''\nx'''"
        s: '\nx'
  ]
--- JSON ---
{
  "contents": [
    {
      "id": "tid_1", 
      "type": "comment", 
      "value": "# DOS-style line endings with \\r\\n to end each line"
    }, 
    "\n", 
    {
      "contents": [
        {
          "contents": [
            {
              "id": "id_0", 
              "type": "Str", 
              "value": "'''\nx'''"
            }
          ], 
          "id": "id_0", 
          "type": "Expr"
        }
      ], 
      "id": "id_0", 
      "type": "Module"
    }, 
    "\n"
  ], 
  "id": "id_ROOT", 
  "type": "ROOT_NODE"
}
